docs/architecture/rewards.md

```diff
@@ -0,0 +1,25 @@
+# Staking Rewards
+
+ZetaChain is a delegated proof of stake blockchain. Holders of ZETA tokens can
+stake their tokens by delegating to validators. Staking tokens allows holders to
+participate in governance and earn rewards for securing the network.
+
+Currently, staking rewards are comprised of transaction fees collected each
+block and fixed rewards coming from the emissions pool.
+
+$$
+StakingRewards = TransactionFees + {EmissionsPool \over NumberOfBlocksPerYear × 4}
+$$
+
+Emissions pool address:
+[zeta1w43fn2ze2wyhu5hfmegr6vp52c3dgn0srdgymy](https://zetachain.blockpi.network/lcd/v1/public/cosmos/auth/v1beta1/module_accounts)
```

Comment:
maybe we can use this query instead , https://zetachain-athens.blockpi.network/lcd/v1/public/zeta-chain/emissions/list_addresses

Reply:
2f2d580